packages/admin/src/Pages/Actions/Action.php

```diff
@@ -78,9 +77,9 @@ public static function makeModalAction(string $name): ModalAction
         return $action;
     }
 
-    public function notify(string | Closure | null $status, string | Closure | null $message): void
+    public function notify(Notification $notification): void
```

Comment:
@danharrin, this would be a breaking change, right?

Reply:
It's also strange that it accepted closures, since `Filament::notify()` did and still does not support that.